lib/joi/LICENSE

```diff
@@ -1,173 +1,83 @@
-===
```

Comment:
The license file was updated because the package and its dependencies underwent some reorganization. The content from each license section is copied verbatim from the corresponding project's repo.
* joi: https://github.com/hapijs/joi/blob/v15.0.3/LICENSE.md
* address: https://github.com/hapijs/address/blob/v2.0.0/LICENSE.md
* hoek: https://github.com/hapijs/hoek/blob/v6.2.3/LICENSE.md
* marker: https://github.com/hapijs/marker/blob/v1.0.0/LICENSE.md
* topo: https://github.com/hapijs/topo/blob/v3.1.1/LICENSE.md

See the "Dependencies" tab on [joi's npm package listing](https://www.npmjs.com/package/@hapi/joi) to verify the list of dependencies is accurate and complete.